### PR TITLE
e2e / nightwatch updates

### PR DESCRIPTION
**Pull request by jakemcdermott**
##### SUMMARY
- update to latest nightwatch and chromedriver versions, unpin chrome and hub container images
- add command to upload files from runner to workers using webdriver wire protocol. This makes tests involving file widgetry runnable in containers.
- a handful of small test fixes and also some changes to help stabilize local smoke test runs

### CODE REVIEW

File: awx/ui/test/e2e/commands/navigateTo.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
-exports.command = function navigateTo (url) {
+const spinny = 'div.spinny';
+
+exports.command = function navigateTo (url, expectSpinny = true) {
+    this.url('data:,'); // https://github.com/nightwatchjs/nightwatch/issues/1724
```
Comment (jakemcdermott) [inline awx/ui/test/e2e/commands/navigateTo.js:4]: There's a minor bug with the built-in url command introduced by the latest nightwatch version, so this workaround/wrapper has been added anywhere it is being used.

File: awx/ui/test/e2e/commands/pushFileToWorker.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { basename } from 'path';
```
Comment (jakemcdermott) [inline awx/ui/test/e2e/commands/pushFileToWorker.js:1]: File widgetry checks are now runnable in containers.

File: awx/ui/test/e2e/objects/activityStream.js [hunk 3]
```diff
@@ -2,6 +2,12 @@ module.exports = {
     url () {
         return `${this.api.globals.launch_url}/#/activity_stream`;
     },
+    commands: [{
+        load () {
```
Comment (jakemcdermott) [inline awx/ui/test/e2e/objects/activityStream.js:6]: ditto https://github.com/ansible/awx/pull/1344/files#r170469810

**Comment by gconsidine** [comment]
I gave it a run and a review. It's solid. :+1:

**Review by gconsidine — APPROVED** [review]

**Pull request merged**